app/dashboard/templates/bounty/fulfill.html

```diff
@@ -126,7 +132,7 @@ <h4 class="text-center mb-4">{% trans "Submit Work" %}</h4>
                   <div class="mt-2 row">
                     <div class="col-12">
                       <label class="form__label" for="hoursWorked">{% trans "Hours Worked" %}</label>
-                      <input name='hoursWorked' id='hoursWorked' class="form__input font-body" type="number" placeholder="1 (feel free to approximate)" value="" min="0" step="0.1" required/>
+                      <input name='hoursWorked' id='hoursWorked' class="form__input font-body" type="number" placeholder="1 (feel free to approximate)" value="" min="0" step="1" required/>
```

Comment:
why do we do this ?

Reply:
because https://github.com/gitcoinco/web/issues/7695

Reply:
we are encouraging decimals on the ui when actually the backend doesnt accept it

Reply:
I think we could watch and do a Math.ceil.

Reply:
do you want to add the backend validation Abdul?

Reply:
right now is `if not hours_worked or not hours_worked.isdigit():`

Reply:
the model does support decimals 
`fulfiller_hours_worked = models.DecimalField(null=True, blank=True, decimal_places=2, max_digits=50)`

Reply:
i know this was a specific requirement from scott a while back

Reply:
mm probably is because the comma vs dot?

Reply:
![image](https://user-images.githubusercontent.com/1358093/96480291-6f3d5800-1210-11eb-8ef9-79300a520461.png)

Reply:
seems the problem is with hours_worked.isdigit()

Reply:
yup that's the issue we have to update that to check if has numbers or . and not characters